src/main.jsx

```diff
@@ -68,15 +68,6 @@ const App = () => {
     document.body.className = theme;
   }, [theme]);
 
-  useEffect(() => {
-    const lenis = new Lenis();
-    function raf(time) {
-      lenis.raf(time);
-      requestAnimationFrame(raf);
-    }
-    requestAnimationFrame(raf);
-  }, []);
-
```

Comment:
This just removes the use of Lenis completely right ?

Reply:
Yes

Reply:
Then let's do one thing. Remove it completely as dependency. Let's keep it very very simple without lenis at all. Remove it from package.json and from imports too.

Reply:
I am on the way to commit Changes.